cloudsync/tasks.py

```diff
@@ -88,7 +88,7 @@ def transcode_from_s3(self, video_id):  # pylint: disable=unused-argument
     } for preset in settings.ET_PRESET_IDS]
 
     playlists = [{
-        'Format': 'HLSv4',
+        'Format': 'HLSv3',
```

Comment:
It is a combination of this and different default presets.  The presets handle both audio and video.  HLSv4 requires sound and video to be in separate presets (so 3 video presets and 1 audio preset), which is fine and I could have just added an additional audio preset to the transcode job, but I think HLSv3 may also be more compatible with older iOS/Android phones.